js/controller.js

```diff
@@ -2061,6 +2057,18 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       return element;
     },
 
+    /**
+     * Adds a blur effect to the video
+     * @protected
+     * @method Html5Skin#addBlur
+     */
+    addBlur: function() {
+      // [PLAYER-2220]: videoVr should not blur. This prevents a circular review on a pause.
+      if (!this.videoVr) {
+        this.state.mainVideoElement.classList.add('oo-blur');
```

Comment:
Do we need null checks for `mainVideoElement` since this can be called at a different point in the code?

Reply:
Added